charts/backbeat-consumer/values.yaml

```diff
@@ -4,7 +4,7 @@
 replicaCount: 1
 image:
   repository: zenko/backbeat
-  tag: pensieve-2
+  tag: 0.1.4
```

Comment:
This will need to change because we will need a new S3 docker image from this S3 PR https://github.com/scality/S3/pull/1170

Reply:
I rebuild the image

Reply:
actually it is cloudserver, not backbeat